### PR TITLE
Migrate delete confirmation for rails 4

### PR DESCRIPTION
**Pull request by hab278**

### CODE REVIEW

File: app/views/forum_groups/index.html.haml [hunk 1]
```diff
@@ -7,7 +7,7 @@
       %span.controls.right.smaller
         = link_to "New Forum", new_forum_path if can? :create, Forum
         = link_to "Edit Group", edit_forum_group_path(group) if can? :manage, group
-        = link_to "Delete Group", forum_group_path(group), :confirm => "Are you sure you want to delete this group?", :method => :delete if can? :manage, group
+        = link_to "Delete Group", forum_group_path(group), :data => { confirm => "Are you sure you want to delete this group?" }, :method => :delete if can? :manage, group
```
Comment (svasandani) [inline app/views/forum_groups/index.html.haml]: missing a colon

**Review by svasandani — CHANGES_REQUESTED** [review]
pls add the colon ty

File: app/views/forum_groups/index.html.haml (hunk 1 above)
Comment (hab278) [reply app/views/forum_groups/index.html.haml]: donezo

**Comment by svasandani** [comment]
dope ty

**Pull request merged**